basket/src/components/Login.tsx

```diff
@@ -0,0 +1,19 @@
+type LoginProps = {
+  login: (id: number, name: string) => void;
+  inputValue: string;
+  setInputValue: React.Dispatch<React.SetStateAction<string>>;
+};
+
+
+export const Login = ({ login, inputValue, setInputValue }: LoginProps) => {
+  const handleLogin = () => {
+    login(1, inputValue);
+  }
+
+  return (
+    <div>
+      <input type="text" value={inputValue} onChange={(e) => setInputValue(e.target.value)} />
+      <button onClick={handleLogin}>Login</button>
```

Comment:
html의 button 태그에 type 속성을 따로 지정해주지 않으면 기본값은 type이 submit으로 인식되기 때문에, 웹 접근성을 위해 `type='button'` 작성해주기!

```suggestion
      <button type='button' onClick={handleLogin}>Login</button>
```